platform/core/src/defaults/hotkeyBindings.js

```diff
@@ -0,0 +1,95 @@
+export default [
```

Comment:
Interesting. I guess these are all technically agnostic of any extension. Putting them in core as defaults works well. Good idea :+1: